scss/bitstyles/settings/breakpoints.stories.mdx

```diff
@@ -0,0 +1,96 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs/blocks';
+
+<Meta title="Design Tokens/Breakpoints" />
+
+# Breakpoints
+
+Bitstyles provides a central Sass map of global media queries that are then used to make almost all utilities, and many components, available limited to specified breakpoints.
+
+Default values are:
+
+```scss
+$breakpoints: (
+  's': 'screen and (max-width: 29.9375em)',
+  'm': 'screen and (min-width: 30em)',
+  'l': 'screen and (min-width: 55em)',
+  'xl': 'screen and (min-width: 95em)',
+  'hidpi':
+    '(-webkit-min-device-pixel-ratio: 2), (min-resolution: 192dpi), (min-resolution: 2dppx)',
+  'landscape': 'all and (orientation:landscape)',
+  'portrait': 'all and (orientation:portrait)',
+  'motion-ok': 'screen and (prefers-reduced-motion: no-preference)',
+) !default;
+```
+
+This setup assumes a mobile-first approach, where base styles are be applied to all viewport sizes, then overridden on larger. Therefore the four size-based breakpoints are specified using `min-width` i.e. they apply to viewport sizes larger than the specified value.
+
+The exception is the `s` breakpoint. As we’re applying styles always mobile-first, we don’t need a small-or-larger breakpoint — that’s what we have already when we don’t use any breakpoint. The `s` breakpoint instead applies **only** to small viewport sizes, and is there for the exceptional circumstances where applying exceptions for small viewport sizes (instead of vice versa) results in simpler code with fewer classnames or CSS properies.
```

Comment:
I reworked this a little, wdyt?:

```
The exception is the `s` breakpoint. As we’re applying styles mobile-first, we don’t need a “small-or-larger” breakpoint — that’s what we have already have when we don’t use any breakpoints. The `s` breakpoint instead applies **only** to small viewport sizes, and is there for the exceptional circumstances where applying overrides for small viewport sizes (instead of vice versa) results in simpler code with fewer classnames or CSS properties.
```